app/shared/data/common/data/models/UISettings.kt

```diff
@@ -48,6 +49,17 @@ data class MyCollectionsSettings(
     }
 }
 
+@Serializable
+@Immutable
+data class MySearchSettings(
```

Comment:
```suggestion
data class SearchSettings(
```

`MyCollections` 实际上是 "我的追番" 的意思